src/Api.js

```diff
@@ -0,0 +1,7 @@
+export const fetchReviews = () => {
+  return fetch("https://nc-games-d5cd.onrender.com/api/reviews")
+    .then((res) => res.json())
+    .then((data) => {
+      return data;
+    });
+};
```

Comment:
it would be good here to have a base URL using `axios.create` then you wouldn't need to be repeating the whole URL in each function and it improves readability 👌